src/components/Player/Pages/Matches/playerMatchesColumns.jsx

```diff
@@ -23,27 +23,46 @@ export default [{
   field: 'skill',
   sortFn: true,
   displayFn: transformations.skill,
+  mediaQ: 'skill_wide',
 }, {
```

Comment:
Here's the example:
```js
mediaQuery: width => width <= 880; 
```
This will give us more flexibility if we want to do stuff more granularly without locking us into these predefined widths. We can also leverage predetermined widths within redux-responsive (or we can define custom ones). Here's the documentation on that. 
https://github.com/AlecAivazis/redux-responsive#using-custom-breakpoints